cypress/integration/smokeTests/sharedTests/auth-tests.ts

```diff
@@ -251,6 +257,25 @@ function testWorkflow(registry: string, repo: string, name: string) {
       cy.contains('button', 'Restub').click();
       cy.contains('button', 'Publish').should('be.disabled');
     });
+    it('hide and un-hide a version', () => {
+      function toggleHidden() {
+        cy.get('[data-cy=refreshButton]').click();
+        cy.get('[data-cy=versionRow]').last().contains('button', 'Actions').should('be.visible').click();
+        cy.contains('button', 'Edit').click();
+        // TODO: Use [data-cy=hiddenCheck] -- do after 1.13 deployed
+        cy.contains('div', 'Hidden:').within(() => {
+          cy.get('[name=checkbox]').click();
+        });
+        cy.contains('button', 'Save Changes').click();
+      }
+      goToTab('Versions');
+      // hide
+      toggleHidden();
+      cy.get('[data-cy=hidden]').should('have.length', 1);
+      // un-hide
+      toggleHidden();
+      cy.get('[data-cy=hidden]').should('not.exist');
```

Comment:
Flynn was suggesting that we set the same value of `data-cy` (`hiddenCheck` is probably good) for both the tools and workflows pages, which sounds like a good idea.

Reply:
I had added the `data-cy=hiddenCheck`, but I don't want to use it here yet because it will not be deployed to staging/prod. I'll create a followup ticket.